assets/styles/_variables.scss

```diff
@@ -14,17 +14,32 @@ $component-spacing-tablet: 80px;
 $component-spacing-mobile: 40px;
 $gutter-width: 24px;
 
-$color-brand: #0d8eff;
-$color-accent: #f96417;
-$color-background: #fff;
-$color-contrast: #414751;
-$color-text: $color-contrast;
-$color-headline: #0b1016;
-$color-border: #d8d8d8;
+:root {
+  --color-border: #d8d8d8; // TODO: Add this on customizer?
+  --color-background: #fff;
+  --color-contrast: #414751;
```

Comment:
I believe the background and contrast colors should also be editable in the customizer.
We wouldn't need an option to change the text color, as it's the same as the contrast color. Or we could get rid of the contrast color and only use the text color.